plugins/modules/elb_network_lb.py

```diff
@@ -405,7 +404,10 @@ def create_or_update_elb(elb_obj):
     # ip address type
     snaked_elb['ip_address_type'] = elb_obj.get_elb_ip_address_type()
 
-    elb_obj.module.exit_json(changed=elb_obj.changed, **snaked_elb)
+    elb_obj.module.exit_json(
+        changed=elb_obj.changed,
+        load_balancer=snaked_elb,
```

Comment:
Not too familiar with this module but is this `load_balancer` return supposed to be here? I couldnt find it in the DOCS section

Reply:
I forgot to update the docs :)

I was planning on nesting it as per https://github.com/ansible-collections/amazon.aws/pull/865 :)